src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/Pages/_Host.cshtml

```diff
@@ -17,6 +17,17 @@
         @(await Html.RenderComponentAsync<App>(RenderMode.ServerPrerendered))
     </app>
 
+    <div id="error-ui">
+        <environment include="Staging,Production">
+            An error has occurred. This application may no longer respond until reloaded.
+        </environment>
+        <environment include="Development">
+            An unhandled exception has occurred. See browser dev tools for details.
+        </environment>
+        <a href="#" class='reload'>Reload</a>
+        <a href="#" class='dismiss'>X</a>
```

Comment:
Also for these, I suggest replacing the `'` with `"` (I know it's awkward we have different conventions for each language), removing the `href`, and replacing `X` with `🗙`.